jwtauth.go

```diff
@@ -76,6 +114,23 @@ func Verify(ja *JWTAuth, findTokenFns ...func(r *http.Request) string) func(http
 	}
 }
 
+func VerifyDynamic(jaf func() (*JWTAuth, error), findTokenFns ...func(r *http.Request) string) func(http.Handler) http.Handler {
```

Comment:
we don't need this method, as we can check which "mode" the JWTAuth object is in by checking verifyKey or keySet

Reply:
The idea behind this functionality is to allow keys to be obtained dynamically, such as in integration with Vault.